src/README.md

```diff
@@ -189,6 +189,40 @@ The process of proving vulnerabilities, is the base concept of the Active Huntin
 To prove a vulnerability, create an `ActiveHunter` that is subscribed to the vulnerability, and inside of the `execute`, specify the `evidence` attribute of the event.   
 *Note that you can specify the 'evidence' attribute without active hunting*  
 
+## Filtering Events
+Sometimes, you may want to filter some events, based on a specific attribute.  
+Creating a filter is simple, the API is similar to creating a Hunter.  
+By inheriting from `EventFilterBase` in `src.core.events.types` module, you can create the filter class.  
+Then, use `@handler.subscribe(Event)` to filter a specific `Event`, kube-hunter's core, will make sure to run your filter on the event before publishing it.   
+#### Similarities to the Hunter API:
+- You need to implement `self.execute(self)` method.
+- to access the event that's being filtered, use `self.event` in your filter.  
+
+#### Options for filtering:
+* Remove/Prevent an event from being published
+* Altering event attributes
+
+To prevent an event from being published, return `True` from the execute method of your filter.  
+To alter event attributes, alter the `self.event`, it will change the event itself before it is published.  
+__make sure to not return anything from the execute method if you just want to alter attributes__  
```

Comment:
If you're altering attributes should you return None? Or False? Returning False seems more like the opposite of True :-)

Reply:
You can return False as well, It's basically the same thing, but not returning anything from a function is equal to returning None.